docker-compose.yaml

```diff
@@ -65,10 +77,22 @@ services:
       - .env.docker.dev
     environment:
       - START_PROCESS=worker
+      - CHAIN_ENVIRONMENT=dev
+      - PROVIDER_ID=1
+      - PROVIDER_ACCOUNT_SEED_PHRASE=//Alice
+      - FILE_UPLOAD_MAX_SIZE_IN_BYTES=10000000 # ~10Mb
+      - ASSET_EXPIRATION_INTERVAL_SECONDS=300
+      - BATCH_INTERVAL_SECONDS=12
+      - BATCH_MAX_COUNT=1000
+      - ASSET_UPLOAD_VERIFICATION_DELAY_SECONDS=5
+      - CAPACITY_LIMIT={"type":"percentage", "value":80}
+      - REDIS_URL=redis://redis:6379
+      - FREQUENCY_URL=ws://frequency:9944
+      - IPFS_ENDPOINT=http://ipfs:5001
```

Comment:
In other services we use an env file here, rather than hard-coding into the compose yaml...

Reply:
This is the only service that "relies" on another service (or is using it for ease of use).

I wasn't sure where to put the env vars that are really just for the dependent service and not the service being developed.

Open to ideas, but it cannot be `env.docker.dev` I don't think? As the publisher service might need different values than the watcher